src/main/java/edu/harvard/h2ms/web/controller/PasswordController.java

```diff
@@ -175,6 +181,18 @@
     // TODO: is there a password policy?
     userRepository.save(user);
 
+    URL ref;
+    try {
+      ref = new URL(referer);
+    } catch (MalformedURLException e) {
+      throw new RuntimeException(e.getMessage());
+    }
+
+    String url =
+        String.format(
+            "%s://%s:%d/reset-password/%s",
+            ref.getProtocol(), ref.getHost(), ref.getPort(), user.getEmail());

```

Comment:
Can we add a TODO to replace this with a callback?

I don't expect it to get done this sprint, but if there is any extra time it would be nice.